### DISCUSSION TITLE
Questions about Common Lisp support

### DISCUSSION THREAD

**Discussion by Kirtai** [Interlisp/medley#280] [Implementation history]
I see that Medley supports Common Lisp in addition to Interlisp.

What is the status of this? Is it ANSI, CLtL1 or 2, or some other variant? (I recognise that it's old though)

What's the difference between the "Common Lisp" and "Xerox Common Lisp" interactors in the sysouts?

And will both CL & Interlisp be supported going forward or just Interlisp?

Sorry if this is documented somewhere but I've been unable to find anything about it other than that it was added in Lyric.

**Comment by masinter** [comment]
short answer: don't know, requires investigation.   See issue #5 

I think the difference between the different execs were just the default package and readtable, but I don't know.

**Comment by Anzus** [comment]
It's a CLtL1, with an optional PCL. There does exist a CLtL2 version, but it hasn't been rebuilt using the latest emulator.
I'll work on uploading the release notes, which describe some of the changes as well as the difference between the various exec's.

**Comment by johnwcowan** [comment]
I consider upgrading to full ANSI CL to be essential if Medley is to be used as an IDE for CL development.

**Comment by masinter** [comment]
I converted this to a Discussion because there isn't any specific action called for.
The implied documentation issue is just another part of what we need to do to document.
Going forward, sure, it would be great to have a great CL ansii standard implementation. I was going to say that it wasn't in anyone's goals in [Medley Project Goals](https://docs.google.com/document/d/1q15mKHJt1fiFToamUacDWvw2UnUDJIaWLuuaeEUQS8k/edit?usp=sharing), but then I saw that it was.
